app/vmagent/remotewrite/remotewrite.go

```diff
@@ -773,7 +773,7 @@ func newRemoteWriteCtx(argIdx int, remoteWriteURL *url.URL, maxInmemoryBlocks in
 	}
 
 	isPQDisabled := disableOnDiskQueue.GetOptionalArg(argIdx)
-	fq := persistentqueue.MustOpenFastQueue(queuePath, sanitizedURL, maxInmemoryBlocks, maxPendingBytes, isPQDisabled)
+	fq := persistentqueue.MustOpenFastQueue(queuePath, remoteWriteURL.String(), maxInmemoryBlocks, maxPendingBytes, isPQDisabled)
```

Comment:
@hagen1778  WDYT of this change? I'd like to keep it sanitizedURL as queue name. 
 Because it'll erase queue content if version of vmagent will be downgraded from new release to previous with default settings.

Reply:
If we drop the check for `name` in `metadata.json`, this change may not be necessary.

Reply:
Lets keep sanitizedURL for backward compatibility.